src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -110,24 +113,27 @@ limitations under the License.
     <item name="Video" value="0x0D"/>
   </enum>
 
-  <enum name="ContentLaunchStatusEnum" type="enum8">
+  <enum name="StatusEnum" type="enum8">
     <cluster code="0x050a"/>
-    <item name="success" value="0x00"/>
-    <item name="urlNotAvailable" value="0x01"/>
-    <item name="authFailed" value="0x02"/>
+    <item name="Success" value="0x00"/>
+    <item name="URLNotAvailable" value="0x01"/>
+    <item name="AuthFailed" value="0x02"/>
+    <item name="TextTrackNotAvailable" value="0x03"/>
+    <item name="AudioTrackNotAvailable" value="0x04"/>
   </enum>
 
   <struct name="ParameterStruct">
     <cluster code="0x050a"/>
     <item name="Type" type="ParameterEnum"/>
-    <item name="Value" type="char_string"/>
+    <item name="Value" type="char_string" length="1024" />
     <item name="ExternalIDList" type="AdditionalInfoStruct" array="true" optional="true"/>
   </struct>
 
-  <bitmap name="SupportedStreamingProtocol" type="bitmap32">
+  <bitmap name="SupportedProtocolsBitmap" type="bitmap32">
     <cluster code="0x050a"/>
     <field name="DASH" mask="0x1"/>
     <field name="HLS" mask="0x2"/>
+    <field name="WebRTC" mask="0x4"/>
```

Comment:
This is 1.3 provisional, yes?  Is the goal here to align with the 1.2 version of the spec, or with the 1.3 proposed-but-not-balloted-yet changes?  

This is going to need some Darwin availability annotation changes, but I'd like to make sure we pin this question down before making them.

Reply:
I just used whatever spec copy I had ... is there a way for me to see 1.3 provisional vs 1.2? Or should I just get some 1.2 spec copy?

Reply:
Will use a 1.2 spec and revert to version 1.2 only (revision 1)